docs/containers/debug-python.md

```diff
@@ -228,6 +228,126 @@ When you select **Docker: Add Docker Files to Workspace** for Django or Flask, w
 1. Modify and save any file.
 1. Refresh the browser and validate changes have been made.
 
+## How to build and run a container together
+
+1. In the previosly mentioned `tasks.json` file we have a dependency on the `docker-build` task. The task is part of the tasks array in the `tasks.json` and can look like this:
+
+```json
+"tasks":
+[
+  {
+    ...
+  },
+  {
+    "label": "docker-build",
+    "type": "docker-build",
+    "dockerBuild": {
+        "context": "${workspaceFolder}",
+        "dockerfile": "${workspaceFolder}/Dockerfile",
+        "tag": "YOUR_IMAGE_NAME:YOUR_IMAGE_TAG"
+    }
+  }
+]
+```
+
+> [!TIP]
+> As the dependency clearly states `docker-build` as its dependency, the name has to be matches in this task. It can be changed to anything though.
+
+1. The `dockerBuild` object in the json allows for the following parameters:
+
+    - context: The docker build context, which your Dockerfile is called from
+    - dockerfile: The path to the Dockerfile to execute
+    - tag: The name of the image to be build, together with its version tag
+
+1. Overall, a vscode setup for building and debugging your Flask application can look like this:
```

Comment:
Please use "VS Code" to refer to the VS Code product.

Reply:
Updated the doc to use VS Code.